src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,145 +1,177 @@
 
-    configureButtonBindings();
+    // ==================OPERATOR CONTROLS======================================
 
-    int pairButton;
-    String hidType = m_driverHID.getHID().getName();
-    if (hidType.equals("")) { // Xbox Controller | Name Unknown
-      m_controllerType = 1;
-      pairButton = 7;
-    } else if (hidType.equals("Wireless Controller")) { // PS5 | Is still called "Wireless Controller" if plugged in with a wire.
-      m_controllerType = 2;
-      pairButton = 7;
-    } else {
-      m_controllerType = 0;
-      pairButton = 7;
-    }
+    // Create Triggers here | Triggers should be named t_CommandName
+    Trigger leftTrigger = m_operatorHID.axisGreaterThan(3, Constants.DEADZONE);
+    Trigger rightTrigger = m_operatorHID.axisGreaterThan(4, Constants.DEADZONE);
+
+    Trigger operatorDriveTrigger = m_operatorHID.axisGreaterThan(1, Constants.DEADZONE);
+    operatorDriveTrigger.onTrue(m_operatorDrive);
+
+    Trigger operatorSpeedUp = m_operatorHID.button(2);
+    Trigger operatorSpeedDown = m_operatorHID.button(3);
+    operatorSpeedUp.onTrue(new setSpeedCommand(true, m_tankDriveSubsystem));
+    operatorSpeedDown.onTrue(new setSpeedCommand(false, m_tankDriveSubsystem));
 
-    Trigger resetControls = m_driverHID.button(pairButton)
-      .whileTrue(new setupControls());
+    Trigger intakeMoveUp = m_operatorHID.axisGreaterThan(1, Constants.DEADZONE);
+    Trigger intakeMoveDown = m_operatorHID.axisLessThan(1, -Constants.DEADZONE);
+    intakeMoveUp.whileTrue(new InstantCommand(() -> m_intake.setRaiseIntakeSpeed(0.1), m_intake));
+    intakeMoveDown.whileTrue(new InstantCommand(() -> m_intake.setRaiseIntakeSpeed(-0.1), m_intake));
+    intakeMoveUp.or(intakeMoveDown).onFalse(new InstantCommand(() -> m_intake.setRaiseIntakeSpeed(0), m_intake));
 
-    
+
+
+    // ================DRIVER CONTROLS==========================================
+    // create commands
+    // 5 = left bumper
+    // 6 = right bumper
+
+    // Intake is toggled when left bumper is pressed
+    Trigger flipTrigger = m_driverHID.button(5);
+    flipTrigger.onTrue(m_intake.new FlipIntake(m_intake));
+
+    // Intake runs FORWARD when right trigger is pressed
+    Trigger runIntakeForwardsTrigger = m_driverHID.axisGreaterThan(3, Constants.DEADZONE);
+    runIntakeForwardsTrigger.whileTrue(new IntakeCommand(m_intake, true));
+
+    // Intake runs BACKWARD when right bumper is pressed
+    Trigger runIntakeBackwardsTrigger = m_driverHID.button(6);
+    runIntakeBackwardsTrigger.whileTrue(new IntakeCommand(m_intake, false));
+
+    Trigger invertDrivetrainTrigger = m_driverHID.button(1)
+    .onTrue(m_tankDriveSubsystem.new SwitchDrivetrainInvert(m_tankDriveSubsystem));
+
+    // new Intake
+    // Intake is reversed when right bumper is pressed
+    SwitchIntakeDirection m_switchIntake = m_intake.new SwitchIntakeDirection(m_intake);
+    Trigger switchIntakeTrigger = m_driverHID.button(6);
+    switchIntakeTrigger.onTrue(m_switchIntake);
```

Comment:
I believe this section, and really the SwitchIntakeDirection command, are both unneeded now. @AkshayKuchibhatla can you confirm and if so chop this out?